Add LICENSE file (#319)
publiccodenet/blog: LICENSE.md

# License

> All content licensed under Creative Commons Zero v1.0 Universal, unless otherwise noted. This may include media, and when licensed otherwise it will have an explicit exception noted where it is used.

Reviewer: ```suggestion
All content licensed under Creative Commons Zero v1.0 Universal, unless stated otherwise. This may include media, in which case it will be accompanied by an explicit exception.
```
